src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/CodeStyleOptions2.cs

```diff
@@ -338,17 +338,18 @@ private static PerLanguageOption2<CodeStyleOption2<ParenthesesPreference>> Creat
 
     internal static class CodeStyleOptionGroups
     {
-        public static readonly OptionGroup CodeStyle = new("", priority: 1);
-
-        public static readonly OptionGroup Usings = new(CompilerExtensionsResources.Organize_usings, priority: 1, parent: CodeStyle);
-        public static readonly OptionGroup ThisOrMe = new(CompilerExtensionsResources.this_dot_and_Me_dot_preferences, priority: 2, parent: CodeStyle);
-        public static readonly OptionGroup PredefinedTypeNameUsage = new(CompilerExtensionsResources.Language_keywords_vs_BCL_types_preferences, priority: 3, parent: CodeStyle);
-        public static readonly OptionGroup Parentheses = new(CompilerExtensionsResources.Parentheses_preferences, priority: 4, parent: CodeStyle);
-        public static readonly OptionGroup Modifier = new(CompilerExtensionsResources.Modifier_preferences, priority: 5, parent: CodeStyle);
-        public static readonly OptionGroup ExpressionLevelPreferences = new(CompilerExtensionsResources.Expression_level_preferences, priority: 6, parent: CodeStyle);
-        public static readonly OptionGroup Field = new(CompilerExtensionsResources.Field_preferences, priority: 7, parent: CodeStyle);
-        public static readonly OptionGroup Parameter = new(CompilerExtensionsResources.Parameter_preferences, priority: 8, parent: CodeStyle);
-        public static readonly OptionGroup Suppressions = new(CompilerExtensionsResources.Suppression_preferences, priority: 9, parent: CodeStyle);
-        public static readonly OptionGroup NewLinePreferences = new(CompilerExtensionsResources.New_line_preferences, priority: 10, parent: CodeStyle);
+        private const string name = "Code Style";
+        public static readonly OptionGroup CodeStyle = new(name: "", description: "", priority: 1);
+
+        public static readonly OptionGroup Usings = new(name, description: CompilerExtensionsResources.Organize_usings, priority: 1, parent: CodeStyle);
```

Comment:
>name [](http://example.com/codeflow?start=56&length=4)

This is odd -- all groups should have unique "qualified" name (that is, unique `parent?.Name + name`).

We should also add a test to `VisualStudioOptionStorageTests` that validates this invariant.

The `CodeStyle` group should have `code_style` name. Then the subgroups should have names like `usings`, `this_or_me`, etc.